### PR TITLE
Add new documentation concerning icons

### PR DESCRIPTION
**Pull request by mririgoyen**
## Proposed change

Icons are a large part of Home Assistant, however we have little information in our official documentation about them. In this PR I have added a dedicated page to frontend icons that provides our users with information on how to find icons, suggest new icons and aliases, and even contribute icons to the MDI project!

Additionally, I have updated the Frontend landing page.

---

This is only a first stage. I would like to expand on the icon documentation by adding another page with a curated list of icons by entity category in the future. However, that will require getting MDI into the docs site (replacing Font Awesome), which I have started to do in another branch.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by frenck** [comment]
Do not close to this one little bot... this is waiting for us to review it

**Comment by mririgoyen** [comment]
@frenck Stale again. Anything I can do to help move this along?

**Comment by frenck** [comment]
still relevant stale bot, don't close :)

**Comment by frenck** [comment]
still not stale bot

File: source/_docs/frontend/icons.markdown
Comment (frenck) [inline source/_docs/frontend/icons.markdown]:
```suggestion
The most common way you can find icons is by using the icon picker built right into Home Assistant. Click into the "Icon" field when customizing an entity and start typing. The list will filter to icons that match your search criteria. You can also scroll through all available icons when the field is empty.

```

File: source/_docs/frontend/icons.markdown
Comment (frenck) [inline source/_docs/frontend/icons.markdown]:
```suggestion
  Not all icons that appear in the MDI Picker Browser Extension may be available in Home Assistant (yet!). While the browser extension is updated as MDI releases new packages, Home Assistant may lag behind until its next release.
```

**Comment by mririgoyen** [comment]
@frenck We just actually deprecated the MDI website and some of the URLs and info here are outdated. Let me update this in a couple hours before you fully merge.

File: source/_docs/frontend/icons.markdown
Comment (frenck) [inline source/_docs/frontend/icons.markdown]:
```suggestion
Sometimes an icon exists, but you aren't able to find it with the terms you were searching for. If this has ever happened to you, please [open an issue with the MDI team to suggest new aliases](https://github.com/Templarian/MaterialDesign/issues/new?assignees=&labels=Alias&template=4_alias.yml) that can be added to existing icons.
```

**Review by frenck — APPROVED** [review]
Thanks, @mririgoyen 👍

**Pull request merged**
